### DISCUSSION TITLE
Compilation error happend when setting -DLOG_LEVEL

### DISCUSSION THREAD

**Discussion by Thermelgy-Repo** [eModbus/eModbus#143] [eModbus applications]
Dear Team,

Thankq for this great library, it is working great for our application.

Recently, I have encounter a compilation error when setting  -DLOG_LEVEL from LOG_LEVEL_CRITICAL to LOG_LEVEL_NONE in PlatformIO. Any help from your side will be great.

```
.pio/libdeps/esp32dev/eModbus/src/ModbusServer.cpp: In member function 'ModbusMessage ModbusServer::localRequest(ModbusMessage)':
.pio/libdeps/esp32dev/eModbus/src/ModbusServer.cpp:114:51: error: 'LOG_C' was not declared in this scope
   LOG_C("Internal problem: should not get here!\n");
                                                   ^
*** [.pio\build\esp32dev\lib312\eModbus\ModbusServer.cpp.o] Error 1
```

**Comment by Miq1** [comment]
That has been fixed a few days ago in the ``master`` branch, so please download a fresh copy.
